run_integration.sh

```diff
@@ -7,7 +7,7 @@ pushd $rootdir
 
 # Reset test harness
 rm -rf test-harness
-git clone --single-branch --branch master https://github.com/algorand/algorand-sdk-testing.git test-harness
+git clone --single-branch --branch zc2c https://github.com/algorand/algorand-sdk-testing.git test-harness
```

Comment:
Need to point back to main branch once https://github.com/algorand/algorand-sdk-testing/pull/156 is merged.